graphstorm-processing/graphstorm_processing/config/config_conversion/gconstruct_converter.py

```diff
@@ -14,7 +14,7 @@
 limitations under the License.
 """
 
-from typing import Any
+from typing import Any, Mapping
```

Comment:
`typing.Mapping` was deprecated in favor of `collections.abc.Mapping` in [Py 3.9](https://docs.python.org/3/library/typing.html#typing.Mapping).



```suggestion
from typing import Any
from collections.abc import Mapping
```